src/modules/swap/ModuleSwap.vue

```diff
@@ -1108,12 +1108,6 @@ export default {
       const findToken = this.actualFromTokens.find(item => {
         if (item.contract === this.defaults.fromToken) return item;
       });
-      if (
```

Comment:
why was this removed? this doesn't seem to be related to the issue.

Reply:
I've added it back However, I've pinpointed the issue to be here in this block of code. I've been trying to understand why the return on the 'if' on line 1111 returns mainTokenDetails based on the check. So for now, i've set it to 'findToken' and seems to fix the issue.